commands/actor.go

```diff
@@ -52,10 +55,35 @@ var actorCmd = &cmds.Command{
 }
 
 var actorLsCmd = &cmds.Command{
+	Options: []cmdkit.Option{
+		cmdkit.StringOption("address", "a", "get a actor by address."),
+	},
 	Run: func(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment) error {
-		results, err := GetPorcelainAPI(env).ActorLs(req.Context)
-		if err != nil {
-			return err
+		addrString, _ := req.Options["address"].(string)
+		var results <-chan state.GetAllActorsResult
+		if len(addrString) > 0 {
+			addr, err := address.NewFromString(addrString)
+			if err != nil {
+				return err
+			}
+			act, err := GetPorcelainAPI(env).ActorGet(req.Context, addr)
```

Comment:
My preference would be to make this a separate command, either on `actor` or `show` as stated elsewhere. I don't think we need to be using a channel for this and adding it as a separate command would clean up that logic.